<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="mainStack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ClosedHandCursor</cursorShape>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
      </property>
      <property name="maxLength">
       <number>-1</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>200</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Enter PIN to login</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>340</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_continue">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>260</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Continue       </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>270</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>270</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>270</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>340</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_0">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>410</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_1">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>200</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>340</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>286</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/images/ok.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/background.png);</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>631</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent</string>
      </property>
      <property name="text">
       <string>Please select app to launch</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>671</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <property name="spacing">
        <number>10</number>
       </property>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
